src/openms/source/ANALYSIS/OPENSWATH/IonMobilityScoring.cpp

```diff
@@ -35,7 +36,7 @@ namespace OpenMS
     for (const auto & im_profile : mobilograms)
     {
       mobilityValues.reserve(mobilityValues.size() + im_profile.size());
```

Comment:
just as a sidenote: this is a pessimization (as opposed to the optimization it's supposed to be :))
The only advantage there is that it may safe a bit of memory...
There is more to be gained in the whole function, but that's not for this PR...